### DISCUSSION TITLE
No more updates within ControlD config?

### DISCUSSION THREAD

**Discussion by pictosun** [yokoffing/Control-D-Config#9] [Q&A]
Hi,

thanks for all of your work and bringing us that much information.

But what about updates for ControlD? 

NextDNS config is updated quite often, but ControlD looks like it is a one time only? Will we see any updates over here, or don't you use ControlD anymore?

**Comment by yokoffing** [comment]
> NextDNS config is updated quite often

It actually isn't. I just recently "refreshed" it with one or two things, but NextDNS hasn't developed any new features in the past two years.

What is missing from Control D Config that you'd like to see?

**Reply by pictosun** [reply]
Thanks for your feedback and sorry for posting it into the wrong category. Didn't know about the discussion section over here.

Overall it looks like you're preferring ControlD, right? Sad to see, that there is no development within NextDNS...

What I do miss or more or less don't get is:
- DNSSEC: Why not using it? NextDNS and AdGuard DNS are pushing into DNSSEC (overall it's a good thing) - as long as you don't use there geo rewrites (if I'm correct).
- What about folders section. Hagezi does have some good folders (see https://github.com/hagezi/dns-blocklists/tree/main/controld) but there's no way to automate the integration (or didn't I find it)
- maybe a little bit more talking about DNSSEC and EDNS Subnet (ECS) as it's an interesting topic for many.

Thanks for all your hard work here.

**Reply by spider1163** [reply]
Hello,

if you use Secure DNS protocols like DNS-over-HTTPS or DNS-over-TLS, DNSSEC provides virtually no value. Your DNS requests cannot be intercepted or spoofed by anyone. 
Also, since Control D is a manipulating DNS resolver (based on your rules) this is conflicting with the whole concept of DNSSEC. 

https://docs.controld.com/docs/disable-dnssec-option

**Reply by yokoffing** [reply]
> @pictosun DNSSEC: Why not using it?

You were asking for more frequent updates, but did you read what was [already there](https://github.com/yokoffing/Control-D-Config?tab=readme-ov-file#disable-dnssec)?

> What about folders section... there's no way to automate the integration.

Correct.
